translations/oe_content-ro.po

```diff
@@ -189,45 +189,3 @@ msgstr "vineri"
 
 msgid "Saturday"
 msgstr "sâmbătă"
-
```

Comment:
For RO I don't see any dot after the short month name, I think we need to remove it (seems outdated).